node/hack/governor/src/index.ts

```diff
@@ -48,6 +50,21 @@ if (fs.existsSync(IncludeFileName)) {
   },
 */
 
+// Get the existing token list to check for any extreme price changes and removed tokens
+var existingTokenPrices = {};
+var existingTokenKeys: string[] = [];
+var newTokenKeys = {};
+
+fs.readFile("../../pkg/governor/generated_mainnet_tokens.go", "utf8", function(_, doc) {
+  var matches = doc.matchAll(/{chain: (?<chain>[0-9]+).+addr: "(?<addr>[0-9a-fA-F]+)".*symbol: "(?<symbol>.*)", coin.*price: (?<price>.*)}.*\n/g);
```

Comment:
HERE BE DRAGONS! HIDE THEE EYES CHILD! IT'S REGEX!!!
![Here be dragons](https://cdn.akamai.steamstatic.com/steam/apps/945680/extras/triton.gif?t=1667390066)

Reply:
Lol yeah it's not my favourite but since we know the format of the file won't change unless we change this script it seemed like the easiest way to do it. But it's delightfully opaque as regex always is